$NetBSD: patch-Makefile,v 1.2 2026/10/30 10:32:52 prlw1 Exp $

- Avoid depending on python to create .pc file
- pass in LDFLAGS for rpath and relro

--- Makefile.orig	2026-10-30 08:23:51.992400861 +0000
+++ Makefile
@@ -84,7 +84,7 @@ endif
 LIBRARIES=$(STATIC_LIBRARIES)
 endif
 
-LIBS=-L$(DIR_BUILD) $(shell echo $(LIBRARIES) | sed -e 's%$(DIR_BUILD)/lib\([^ ]*\)\.\(a\|so\)%-l\1%g') -lm
+LIBS=-L$(DIR_BUILD) $(LDFLAGS) $(shell echo $(LIBRARIES) | sed -e 's%$(DIR_BUILD)/lib\([^ ]*\)\.\(a\|so\)%-l\1%g') -lm
 
 ##--------------------------------
 ##
@@ -206,10 +206,10 @@ $(DIR_BUILD)/libjxr.pc: $(DIR_SRC)/libjxr.pc.in
 $(DIR_BUILD)/libjxr.pc: $(DIR_SRC)/libjxr.pc.in
 	@python -c 'import os; d = { "DIR_INSTALL": "$(DIR_INSTALL)", "JXR_VERSION": "$(JXR_VERSION)", "JXR_ENDIAN": "$(ENDIANFLAG)" }; fin = open("$<", "r"); fout = open("$@", "w+"); fout.writelines( [ l % d for l in fin.readlines()])'
 
-install: all $(DIR_BUILD)/libjxr.pc
+install: all
 	install -d $(DIR_INSTALL)/lib/pkgconfig $(DIR_INSTALL)/bin $(DIR_INSTALL)/include/libjxr/common  $(DIR_INSTALL)/include/libjxr/image/x86 $(DIR_INSTALL)/include/libjxr/glue $(DIR_INSTALL)/include/libjxr/test $(DIR_INSTALL)/share/doc/jxr-$(JXR_VERSION)
 	install $(LIBRARIES) $(DIR_INSTALL)/lib
-	install -m 644 $(DIR_BUILD)/libjxr.pc $(DIR_INSTALL)/lib/pkgconfig
+	install -m 644 $(DIR_SRC)/libjxr.pc.in $(DIR_INSTALL)/lib/pkgconfig/libjxr.pc
 	install $(DIR_BUILD)/$(ENCAPP) $(DIR_BUILD)/$(DECAPP) $(DIR_INSTALL)/bin
 	install -m 644 $(DIR_SRC)/common/include/*.h $(DIR_INSTALL)/include/libjxr/common
 	install -m 644 $(DIR_SRC)/image/x86/*.h $(DIR_INSTALL)/include/libjxr/image/x86
